web/stories/testgrid-grid-display.stories.ts

```diff
@@ -0,0 +1,26 @@
+import { html, TemplateResult } from 'lit';
+import '../src/testgrid-grid-display';
+
+export default {
+  title: 'Grid',
+  component: 'testgrid-grid-display',
+};
+
+interface Story<T> {
+  (args: T): TemplateResult;
+  args?: T;
+}
+
+interface Args {
+  dashboardName: string;
+  tabName: String;
+}
+
+const Template: Story<Args> = ({
+  dashboardName = '', tabName = '',
+}: Args) => {
+  return html`<testgrid-grid-display .dashboardName="${dashboardName}" .tabName="${tabName}"></testgrid-grid-display>`;
+};
+
+export const Grid = Template.bind({});
+Grid.args = {dashboardName: 'sig-testing-misc', tabName: 'branchprotector'}
```

Comment:
once we have local API set up, we will have an option of faking those

Reply:
Thanks for the fake! Working on making this work atm (rest of the PR should be okay for review).

Reply:
Thanks for the help! Fake works now (I have no idea how I was overcomplicating it 😅 )